utility/test/module_test.go

```diff
@@ -89,8 +88,8 @@ func NewTestingUtilityContext(t *testing.T, height int64) utility.UtilityContext
 	})
 
 	return utility.UtilityContext{
-		Height:  height,
-		Mempool: mempool,
+		Bus:    testUtilityMod.GetBus(), // TODO
```

Comment:
This is what I had in mind 

https://github.com/pokt-network/pocket/pull/437/commits/e9e86c3667dab0234bbde7f7d7f135d89f520850

so that bus can be unexported